setup.py

```diff
@@ -62,6 +71,58 @@ def _is_package_installed(package_name: str) -> bool:
     or "triton"
 )
 
+
+# --------------------------- flagems c extension ------------------------
+library_name = "flag_gems"
+
+
+def get_extensions():
+    debug_mode = os.getenv("DEBUG", "0") == "1"
+    use_cuda = os.getenv("USE_CUDA", "1") == "1"
+    if debug_mode:
+        print("Compiling in debug mode")
+
+    use_cuda = use_cuda and torch.cuda.is_available() and CUDA_HOME is not None
```

Comment:
Expect using torch_device_fn.is_available() instead of  torch.cuda.is_available() 
and torch_device_fn from（from ..runtime import torch_device_fn）

Reply:
Since it is a part of `flag_gems`, it may not use the utilities provided by `flag_gems`. So I removed cuda extensions since it is planned to use c++-only extensions here.